icons/dessert.svg

```diff
@@ -0,0 +1,18 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+ <path d="M12 17c5.417 0 7.5-2.686 7.5-6h-15c0 3.314 2.083 6 7.5 6Z"/>
+ <path d="M8 21h8"/>
+ <path d="M12 18v2"/>
+ <path d="M5.1 10.8a3.454 3.454 0 0 1-.1-.828C5 8.055 6.567 6.5 8.5 6.5S12 8.055 12 9.972c0 .285-.035.563-.1.828"/>
+ <path d="M12.1 10.8a3.455 3.455 0 0 1-.1-.828C12 8.055 13.567 6.5 15.5 6.5S19 8.055 19 9.972c0 .285-.035.563-.1.828"/>
+ <path d="M15.5 6.5a3.5 3.5 0 1 0-7 0"/>
```

Comment:
This one also has some misaligned lines: 
![image](https://user-images.githubusercontent.com/17746067/199506663-4ec9f535-0358-42a1-bcc2-22137f89f5fc.png)

I would also maybe make the bowl more elliptical and add an extra 1px width to it so that it's more pronounced on 100%:
![image](https://user-images.githubusercontent.com/17746067/199507275-a92f7eba-fb8d-4d85-931c-67221532250d.png)
![ice-cream-2](https://user-images.githubusercontent.com/17746067/199507285-5353d0e0-2eb9-4adb-8f8e-44e6ea1e3f42.png)

Also, maybe let's call this `ice-cream-2` or `ice-cream-bowl` instead, "dessert" is maybe a tad too generic.